magento/magento2
# Don't skip row on import if image not available.

### Description (*)
In this commit d4d34ff830c6d65f8803536f950e3edd51ed36bf was added feature to skip rows on this condition (if empty UrlKey or if urlKey has duplicate, if attribute set is not valid , or if sku is empty) but we don't need to skip row if image was not available.

### Fixed Issues (if relevant)

1. magento/magento2#22355: Issue title

### Manual testing scenarios (*)
`Csv file to upload product, with column base_image and his value This image have to be not exist in folder`

1. Go to admin (System - Import - Products)
2. Upload csv
3. Check data
4. Import

### Contribution checklist (*)
